src/main/c/org_apache_activemq_artemis_nativo_jlibaio_LibaioContext.c

```diff
@@ -140,7 +136,7 @@ static int ringio_get_events(io_context_t aio_ctx, long min_nr, long max,
             }
```

Comment:
I would like to get rid of the heavyweight % instruction up there, but that's another story